mavros/src/lib/ftf_frame_conversions.cpp

```diff
@@ -21,21 +21,35 @@
 namespace mavros {
 namespace ftf {
 namespace detail {
-
-// Static quaternion needed for rotating between ENU and NED frames
-// +PI rotation around X (North) axis follwed by +PI/2 rotation about Z (Down)
-// gives the ENU frame.  Similarly, a +PI rotation about X (East) followed by
-// a +PI/2 roation about Z (Up) gives the NED frame.
+/**
+ * @brief Static quaternion needed for rotating between ENU and NED frames
+ * +PI rotation around X (North) axis follwed by +PI/2 rotation about Z (Down)
+ * gives the ENU frame.  Similarly, a +PI rotation about X (East) followed by
+ * a +PI/2 roation about Z (Up) gives the NED frame.
+ */
 static const Eigen::Quaterniond NED_ENU_Q = quaternion_from_rpy(M_PI, 0.0, M_PI_2);
 
-// Static quaternion needed for rotating between aircraft and base_link frames
-// +PI rotation around X (Forward) axis transforms from Forward, Right, Down (aircraft)
-// Fto Forward, Left, Up (base_link) frames.
+/**
+ * @brief Static quaternion needed for rotating between aircraft and base_link frames
+ * +PI rotation around X (Forward) axis transforms from Forward, Right, Down (aircraft)
+ * Fto Forward, Left, Up (base_link) frames.
+ */
 static const Eigen::Quaterniond AIRCRAFT_BASELINK_Q = quaternion_from_rpy(M_PI, 0.0, 0.0);
 
 static const Eigen::Affine3d NED_ENU_AFFINE(NED_ENU_Q);
 static const Eigen::Affine3d AIRCRAFT_BASELINK_AFFINE(AIRCRAFT_BASELINK_Q);
 
+/**
+ * @brief 3-D matrices to fill 6-D rotation matrix applied to change covariance matrices coordinate frames
+ */
+static const Eigen::Matrix3d R_NE = NED_ENU_Q.normalized().toRotationMatrix();
+static const Eigen::Matrix3d R_AB = AIRCRAFT_BASELINK_Q.normalized().toRotationMatrix();
+
+/**
+ * @brief 3-D matrix, filled with zeros, that reprensent top-right and bottom-left of the rotation matrix
+ */
+static const auto zerom3d = Eigen::Matrix3d::Zero();
```

Comment:
I think you leave that inside function scope.
Outside we perhaps better to keep prefixes, `NED_ENU_R`, `AIRCRAFT_BASELINK_R`.

Also capitalize zerom3d.